### ISSUE TITLE
Failed to load app in a work profile in Android

### ISSUE THREAD

**Issue by qnighy** [facebook/flipper#953]
## 🐛 Bug Report

When I launch an Android app in a [work profile](https://developer.android.com/work/managed-profiles), flipper shows the following error:

```
Client setup error: No matching device found for app: com.foo.bar.some_app
```

The app `com.foo.bar.some_app` comes with flipper integration. 

Several months ago I had dug deep into the internals of flipper and it's most likely caused by work profile.

## To Reproduce

- Install a flipper-integrated Android app in a work profile.
- Launch a flipper and connect the Android device to it.

## Environment

- Host device:
  - OS: Mac OS Catalina 10.15.3
  - Flipper: 0.35.0
  - adb: `Android Debug Bridge version 1.0.41` / `Version 29.0.6-6198805`
- Android device:
  - Device: Pixel 4
  - OS: Android 10

**Comment by passy** [comment]
Thanks for the report, @qnighy. I believe that's the first time I hear about work profiles, so it's sadly not entirely unexpected that this doesn't work right out of the box. Unfortunately, this is probably not something we'll have the bandwidth to work on internally, but if you or somebody else would like to pick this up, we'd be delighted to review the PR.

**Comment by sbatezat** [comment]
Hi there,
Is it possible to re-open this issue as I'll shortly submit a PR that is fixing it?
Cheers

**Comment by mweststrate** [comment]
nvm, leaving it closed since #4605 already exists :)
